docs/assets/includes/ai-info.md

```diff
@@ -0,0 +1,5 @@
+!!! note
+    -   This feature is compatible with most programming languages and requires no additional setup.
+    -   Suggestions are generated using the description of the static analysis issue and a few lines of surrounding code.
```

Comment:
I just saw [this comment](https://codacy.slite.com/app/docs/NotkMtv3mJCayS?threadId=JDLQXyMR1DWV99) by @RaquelHipolito and I agree that it's a good idea to explain why we need to share these lines of code when sending the prompt to OpenAI. :ok_hand:

Reply:
Good idea. I updated the admonition to clarify this.